tchannel/tornado/dispatch.py

```diff
@@ -220,20 +222,22 @@ def handle_call(self, request, connection):
 
             response.flush()
         except TChannelError as e:
-            connection.send_error(
-                e.code,
-                e.message,
-                request.id,
-            )
+            e.tracing = request.tracing
+            e.id = request.id
+            connection.send_error(e)
         except Exception as e:
-            msg = "An unexpected error has occurred from the handler"
-            log.exception(msg)
+            log.exception("Unexpected Error: {0}".format(e.message))
```

Comment:
We've had this discussion before. I'm not aware of any concrete reasons why one is better than the other. 

The reason I'm asking for the change here is solely for consistency.